app/web/index.html

```diff
@@ -35,6 +35,10 @@
     <!-- Styles-->
     <link rel="stylesheet" href="index.css" type="text/css" media="screen,projection">
     <script src="assets/packages/fast_rsa/web/assets/wasm_exec.js" type="application/javascript"></script>
+
+    <!-- Package "mobile_scanner" depends on "jsQR", see
+    https://pub.dev/packages/mobile_scanner#web -->
+    <script src="https://cdn.jsdelivr.net/npm/jsqr@1.4.0/dist/jsQR.min.js"></script>
```

Comment:
I think we should download the `jsQR.min.js`, upload it to Firebase Storage and use instead the Firebase Storage URL to avoid calling a third party service, right? Should be better from a privacy & stability perspective.

Reply:
Yeah that would probably be better i guess